code/game/objects/effects/effects.dm

```diff
@@ -63,3 +63,57 @@
 
 /obj/effect/dummy/singularity_act()
 	return
+
+/// Proc used for 96x96 effects such as runes(Nar-sie, Apocalypse, Heretic and Wizard ones). So they don't overlay above the items.
+/obj/effect/proc/overlay_for_96x96_effects()
+	var/mutable_appearance/bottom = mutable_appearance(icon, icon_state)
+	bottom.pixel_x = pixel_x
+	bottom.pixel_w = -(pixel_x)
+	bottom.add_filter("mask", 1, alpha_mask_filter(icon = icon(icon, "row_mask_bottom")))
+	add_overlay(bottom)
+	var/mutable_appearance/middle = mutable_appearance(icon, icon_state)
+	middle.pixel_x = pixel_x
+	middle.pixel_w = -(pixel_x)
+	// Shift physical position up a bit
+	middle.pixel_y = -(pixel_y)
+	middle.pixel_z = pixel_y
+	// Mask out everything but the middle
+	middle.add_filter("mask", 1, alpha_mask_filter(icon = icon(icon, "row_mask_middle")))
+	add_overlay(middle)
+	var/mutable_appearance/top = mutable_appearance(icon, icon_state)
+	top.pixel_x = pixel_x
+	top.pixel_w = -(pixel_x)
+	// Shift physical position up a bit
+	top.pixel_y = -(pixel_y*2)
+	top.pixel_z = (pixel_y*2)
+	// Mask out everything but the top
+	top.add_filter("mask", 1, alpha_mask_filter(icon = icon(icon, "row_mask_top")))
+	add_overlay(top)
+
+	icon_state = ""
```

Comment:
it'd be good to explicitly use the 96x96 dmi here, and also potentially move this proc down to /atom so it can be used elsewhere in future.
It could be further generalized to not care about size in future, but that doesn't need to happen here.

Reply:
I think `icon = 'icons/effects/96x96.dmi'` can runtime something but i have no idea what to check it on.